src/MPQEditor/MPQCircleSelector.ts

```diff
@@ -213,14 +214,49 @@ export class MPQSelectionPanel
   }
 
   public dispose() {
-    //TODO
+    if (!this._closedByOwner && this._mpqEventHandler) {
+      this._mpqEventHandler.onClosed(this._ownerPanel);
+    }
+
+    this.disposeGraphCtrl();
+
+    MPQSelectionPanel.panels.forEach((selPan, index) => {
+      if (
+        this._ownerId === selPan._ownerId &&
+        this._modelPath === selPan._modelPath
+      ) {
+        MPQSelectionPanel.panels.splice(index, 1);
+      }
+    });
+
+    // Clean up our resources
+    this._panel.dispose();
+
+    while (this._disposables.length) {
+      const x = this._disposables.pop();
+      if (x) {
+        x.dispose();
+      }
+    }
   }
 
   /**
    * CircleGraphEvent interface implementations
    */
-  public onViewMessage(_message: any) {
-    // TODO
+  public onViewMessage(message: any) {
+    switch (message.command) {
+      case MessageDefs.selection:
```

Comment:
I've got some UI feedbacks. If it's not easy to handle on this pr, you may consider it later.

![image](https://user-images.githubusercontent.com/17171963/235035148-9c4d50f4-c34b-442c-9984-cc08a09ea378.png)

On #1505 draft, 
- when I click on 'add' node, MPQEditor seems to adds the node to the mpq editor slot. 
- when I click on 'Place holder's or 'Add 2' node, as they are not the `mpq` target, they shows the property view on the right side of the circle editor.

I think that 'click' event must result in the same behavior. 
We can choose either to open the property view same to 'add' operation or not to open the property view to other operations.

Reply:
@dayo09 
Thank you. I'll try to fix it in the draft at first.

Reply:
@dayo09 @seanshpark 
The same behaviour in the netron:
 click on input/output nodes shows `Model Properties`, while clicking on the internal nodes show `Nodes Properties`.
For MPQ draft clicking internal nodes results in their selection for editing, while for input/output nodes still shows `Model Properties` to observe input/output size and theirs type.
To handle this issue:
1. We can hide input/output nodes
2. We can disable clicking input/output nodes

But still there may be some issues that some users are used to observe input/output size.
IMHO.

Reply:
Or 3. We can show `Properties` too, when clicking on the internal nodes(`ex. Add`).

Reply:
I will approve this PR, let's go on about it on the other issue or PR. Thanks for listening to me.!

Reply:
> I will approve this PR, let's go on about it on the other issue or PR. Thanks for listening to me.!

Thank you very much!